### ISSUE TITLE
Issue with Transparency on Gifs

### ISSUE THREAD

**Issue by SwiftySubaki** [aseprite/aseprite#1438]
I am attempting to create an animated sprite for my friend; one that uses transparency in one of the layers. 
![sans](https://cloud.githubusercontent.com/assets/26497992/24069248/95bea69c-0b60-11e7-8c19-5ab001eda8c1.png)
This is the sprite normally, when it's not a gif. The flame is fine.
But when I turn it into a animated gif
![sans](https://cloud.githubusercontent.com/assets/26497992/24069256/ba2145da-0b60-11e7-91ab-5af783d7250a.gif)
The flame (and my signature) completely disappears. 

I'd appreciate some help on this; (as the transparency issue appeared on another one of my sprites as well)

Thank you.

### Aseprite and System version

* Aseprite version: v1.1.13, Steam
* System: Windows 10

**Comment by Scrxtchy** [comment]
gifs don't have support for rgba values
you get up to 256 rgb colours, and you can use one of those a single transparency value

This is what it looks like in Photoshop's exporter, which should hopefully visualise this for you
![](https://tenryuu.blob.core.windows.net/astrid/17-03-27_22-25-09.png)

**Comment by Gasparoken** [comment]
@Scrxtchy you are right. Aseprite takes a simple choice to determine if a color should be rendered as transparent or opaque:
  If alpha < 128 ? then the pixel color is assumed transparent;
  else: the pixel is opaque.
Maybe some alternative can be added:
- Is alpha  > configurable threshold ? then a color is assumed opaque.
- Dithering between mask and an opaque colors to simulate colors with alpha < 255 (but maybe is too much).
